Microsoft.Toolkit.Uwp/IncrementalLoadingCollection/IIncrementalSource.cs

```diff
@@ -0,0 +1,47 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Toolkit.Uwp
+{
+    /// <summary>
+    /// This class represents a data source which elements can be loaded incrementally.
+    /// </summary>
+    /// <typeparam name="TSource">Type of collection element.</typeparam>
+    /// <seealso cref="IncrementalLoadingCollection{TSource, IType}"/>
+    public interface IIncrementalSource<TSource>
+    {
+        /// <summary>
+        /// Retrieves items based on <paramref name="pageIndex"/> and <paramref name="pageSize"/> arguments.
+        /// </summary>
+        /// <param name="pageIndex">
+        /// The zero-based index of the page that corresponds to the items to retrieve.
+        /// </param>
+        /// <param name="pageSize">
+        /// The number of <typeparamref name="TSource"/> items to retrieve for the specified <paramref name="pageIndex"/>.
+        /// </param>
+        /// <param name="cancellationToken">
+        /// Used to propagate notification that operation should be canceled.
+        /// </param>
+        /// <returns>
+        /// Returns a collection of <typeparamref name="TSource"/>.
+        /// </returns>
+        Task<IEnumerable<TSource>> GetPagedItemsAsync(int pageIndex, int pageSize, CancellationToken? cancellationToken = null);
```

Comment:
This should really be:

CancellationToken cancellationToken = default(CancellationToken) 

rather than null.